### PR TITLE
added to main.js

### PR DESCRIPTION
**Pull request by mattlub**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 document.querySelector('form').addEventListener('submit', function(event){
   event.preventDefault();
   var searchQuery  = document.querySelector('input').value;
+
   Request.makeRequest(
     'GET',
-    'http://www.omdbapi.com/?t='+searchQuery,
+    'http://www.omdbapi.com/?t=' + searchQuery,
```
Comment (RQ16) [inline main.js:7]: What did that change exactly? I need to know before I decide to merge it

Comment (mattlub) [reply main.js:7]: readability, consistency

Comment (Karyum) [reply main.js:7]: khawa arbadi

Comment (mattlub) [reply main.js:7]: hedi habibi

Comment (mattlub) [reply main.js:7]: haa damajta ha?

Comment (Karyum) [reply main.js:7]: reem lesh enti hmara

Comment (RQ16) [reply main.js:7]: Badeesh aa'mal merge... ento ma btistahalo

**Pull request merged**